packages/flutter_test/lib/src/binding.dart

```diff
@@ -1271,12 +1294,13 @@ enum LiveTestWidgetsFlutterBindingFramePolicy {
   /// This is used for running the test on a device, where scheduling of new
   /// frames respects what the engine and the device needed.
   ///
-  /// Compared to `fullyLive` this policy ignores the frame requests from pump
+  /// Compared to [fullyLive] this policy ignores the frame requests from pump
   /// of the test code so that the frame scheduling respects the situation of
-  /// that for the real environment, and avoids waiting for the new frame beyond
-  /// the expected time.
+  /// that for the real environment, and avoids waiting for the new frame when
```

Comment:
Nit: `new frame` -> `artificially [pump]ed frame`?

Reply:
`new frame` is the original version. But I agree this is more clear. Also leave it open for visibility.